client/components/Home/index.tsx

```diff
@@ -1,32 +1,85 @@
-import React, { ReactElement } from "react";
-import Typography from "@material-ui/core/Typography";
-
+/* eslint-disable prettier/prettier */
+import React, { ReactElement, useState } from "react";
+import { Link } from "react-router-dom";
+import Axios from "axios";
 import { useHomeStyles } from "./styles";
 
-import Feed from "./Feed";
-import RecentVideos from "./RecentVideos";
-import Film from "./FeaturedMovie";
-
 const Home = (): ReactElement => {
   const classes = useHomeStyles({});
+  const [item, setItem] = useState(null);
+  const [refresh, setRefresh] = useState(true);
+  const [itemReady, setItemReady] = useState(false);
+
+  const getRecommendation = (): void => {
+    const list: Array<object> = [];
```

Comment:
the type `object` is not very precise. You can create your own custom type with an interface like that:

```typescript
interface Movie {
  title: string;
  img: string;
  imdbId: string;
  genres: string[];
}
```

All of our general interface are in the `client/models/models.ts` file. You can put it here.

Then You can also type you `item` variable like that:
```typescript
const [item, setItem] = useState<Movie[]>([]);
```